tests/models/state/BaseStateProfile-test.js

```diff
@@ -62,7 +63,7 @@ describe('BaseStateProfile', () => {
     });
     describe('State flag image', () => {
         it('should determine the filename based on FIPS', () => {
-            expect(state.flag).toEqual('https://s3-us-gov-west-1.amazonaws.com/da-public-files/usaspending_state_flags/06.png');
+            expect(state.flag).toEqual(`https://files${kGlobalConstants.DEV ? '-nonprod' : ''}.usaspending.gov/state-flags/06.png`);
```

Comment:
@ebdabbs Same thing with state flags here. In general, the convention for URL mapping seems to follow `lower_case_with_underscores`. 

If you think this is bad convention from a frontend perspective, you could talk it over with @willkjackson. It wouldn't be hard to change but should be changed consistently everywhere.

Probably, to avoid confusion it should follow whatever the bucket naming convention is (underscores or hyphens), so that people don't mistakenly forget translate from one to another.

Reply:
Just a typo - underscores are good with me. We use `lower_case_with_underscores` in frontend urls too.